examples/experimental/dagster-airlift/dagster_airlift_tests/integration_tests/materialize_assets_operator_test_project/dags/dag.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from datetime import datetime
+
+from airflow import DAG
+from dagster_airlift.in_airflow.materialize_assets_operator import DefaultMaterializeAssetsOperator
+
+logging.basicConfig()
+logging.getLogger().setLevel(logging.INFO)
+requests_log = logging.getLogger("requests.packages.urllib3")
+requests_log.setLevel(logging.INFO)
+requests_log.propagate = True
+
+
+default_args = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "retries": 1,
+}
+
+dag = DAG(
+    "the_dag",
+    default_args=default_args,
+    schedule_interval=None,
+    is_paused_upon_creation=False,
+    start_date=datetime(2023, 1, 1),
+)
+the_task = DefaultMaterializeAssetsOperator(
```

Comment:
`Default` is a bit of an odd prefix for a operator we expect to instantiate directly